python/GafferUI/NameValuePlugValueWidget.py

```diff
@@ -51,14 +53,25 @@ class NameValuePlugValueWidget( GafferUI.PlugValueWidget ) :
 
 	def __init__( self, childPlug ) :
 
+		if isinstance( childPlug, Gaffer.Plug ) :
+			childPlugs = { childPlug }
+		else :
+			childPlugs = childPlug or set()
```

Comment:
Hopefully we can also delegate this to the PlugValueWidget constructor and just retrieve with `self.getPlugs()` afterwards?

Reply:
Improved [here](https://github.com/GafferHQ/gaffer/pull/3980/commits/141511f1f746c37498b2b4b0be5fd7360f1c56e5).